src/main.cpp

```diff
@@ -1005,6 +1005,14 @@ int main(int argc, char **args)
   programState.activeViewportIndex = 0;
   programState.pActiveViewport = &programState.pViewports[programState.activeViewportIndex];
 
+  programState.supportedVideoFPSs.Init(16);
```

Comment:
Use a static array at the top of the `vcFlythrough.cpp` file, it's what we do with similar UI elements. What you have done here is incredibly unnecessary.